static/src/javascripts/projects/common/modules/commercial/third-party-tags.js

```diff
@@ -63,6 +63,7 @@ define([
         remarketing.load();
         outbrain.load();
         krux.load();
+        gravity.getRecommendations();
```

Comment:
Yes I put it in the loadOther group so that it behaves in the same way as outbrain.